src/App.js

```diff
@@ -52,12 +56,12 @@ function App() {
     } else {
       setLocalStorage([...localStorage, { ...card, amount: 1 }]);
     }
-    navigate("/warenkorb");
+    /* navigate("/warenkorb"); */
```

Comment:
I think you should delete all code that you don't use, so that your code is 'clean'

Reply:
already done, thank you!